Automattic/jetpack
# Protect: update Protect icon

Fixes #

#### Changes proposed in this Pull Request:

*

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Storybook -> Plugins / Protect / Product Offer
* Storybook -> Plugins / Protect / Interstitial
* Interstitial page
* Summary section


<img width="561" alt="image" src="https://user-images.githubusercontent.com/77539/165765769-76cdc319-ed27-40ad-80bb-056b80f592c2.png">

<img width="734" alt="Screen Shot 2022-04-28 at 11 06 49 AM" src="https://user-images.githubusercontent.com/77539/165771011-1fd6c1ea-6483-40a5-9412-339dd6873220.png">


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202117981194303
